src/encodings.rs

```diff
@@ -15,22 +15,23 @@ use radix_engine_toolkit::functions::scrypto_sbor::{
 #[allow(clippy::redundant_closure)]
 pub fn decode_programmatic_json<T: ScryptoDecode>(
     data: &serde_json::Value,
-) -> Result<T, ScryptoSborError> {
+) -> anyhow::Result<T> {
```

Comment:
these functions depend on radix-engine-toolkit, which hasn't updated SBOR dependencies yet. Since that means it's not the same ScryptoSborError, I had to resort to anyhow errors here

Reply:
Alright, I see. Makes sense